CONTRIBUTING.md

```diff
@@ -81,4 +81,22 @@ and run the tests locally, make sure all tests passed.
 
 ```shell
 make test
-```
\ No newline at end of file
+```
+
+## How to release
+This section guides committers and PMC members to release SkyWalking CLI in Apache Way.
+
+### Prerequisites
+- [x] [GNU Make](https://www.gnu.org/software/make/manual/make.html) is installed
+- [x] [GPG tool](https://gpgtools.org) is installed
+- [x] [Add your GPG key](https://github.com/apache/skywalking/blob/master/docs/en/guides/How-to-release.md#add-your-gpg-public-key)
+
+### Release steps
+- Export the version that is to be released, `export VERSION=1.0.1 `
+- Tag the latest commit that is to be released with `git tag v${VERSION}` and push the tag with `git push https://github.com/apache/skywalking-cli v${VERSION}`
+- Verify licenses, build and sign distribution packages, simply run `make release`, distribution packages and checksums are generated
+- [Upload the packages to SVN repository](https://github.com/apache/skywalking/blob/master/docs/en/guides/How-to-release.md#upload-to-apache-svn) 
+- [Send internal announcement](https://github.com/apache/skywalking/blob/master/docs/en/guides/How-to-release.md#make-the-internal-announcements)
+- [Wait at least 48 hours for test responses](https://github.com/apache/skywalking/blob/master/docs/en/guides/How-to-release.md#wait-at-least-48-hours-for-test-responses)
+- [Call for vote](https://github.com/apache/skywalking/blob/master/docs/en/guides/How-to-release.md#call-a-vote-in-dev)
+- [Publish release](https://github.com/apache/skywalking/blob/master/docs/en/guides/How-to-release.md#publish-release)
```

Comment:
Also, you ref the main doc, it isn't totally right, Such as the svn path, mail contents. I suggest you copy them and change them right.